src/components/comments/Comment.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '../styles/Comment.css';
+
+function Comment({ body, email }) {
+  return (
+    <div className={styles.Comment}>
+      <p>{body}</p>
+      <h4>-{email}</h4>
```

Comment:
```suggestion
      <p>-{email}</p>
```
Typically, it's not best practice to style text with heading tags.